src/samples/UnAuthChildBenefitsClaim/UaUserPortal.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect } from 'react';
+import Button from '../../components/BaseComponents/Button/Button';
+import { useTranslation } from 'react-i18next';
+import useHMRCExternalLinks from '../../components/helpers/hooks/HMRCExternalLinks';
+import setPageTitle from '../../components/helpers/setPageTitleHelpers';
+import NotificationBanner from '../../components/BaseComponents/NotificationBanner/NotificationBanner';
+
+export default function UaUserPortal(props) {
+  const { beginClaim, showPortalBanner } = props;
+  const { t } = useTranslation();
+  const { referrerURL, hmrcURL } = useHMRCExternalLinks();
+
+  useEffect(() => {
+    setPageTitle();
+  }, []);
+
+  return (
+    <>
+      <main className='govuk-main-wrapper' id='main-content' role='main'>
+        {showPortalBanner && (
+          <NotificationBanner content={t('PORTAL_NOTIFICATION_BANNER_CONTENT')} />
+        )}
+        <div className='govuk-grid-row'>
+          <div className='govuk-grid-column-two-thirds'>
+            {/* <h1 className='govuk-heading-xl'>{t('YOUR_CLAIM_APPLICATIONS')}</h1> */}
+            <h1 className='govuk-heading-xl'>Your UnAuth Claim App</h1>
+          </div>
+        </div>
+        <div className='govuk-grid-row'>
+          <div className='govuk-grid-column-full govuk-prototype-kit-common-templates-mainstream-guide-body'>
+            <p className='govuk-body'>{t('THIS_PAGE_SHOWS')}</p>
+            <ul className='govuk-list govuk-list--bullet'>
+              <li>
+                <span className='govuk-body'>{t('CLAIMS_THAT_ARE_IN_PROGRESS')}</span>
+              </li>
+              <li>
+                <span className='govuk-body'>{t('CLAIMS_THAT_ARE_SUBMITTED')}</span>
+              </li>
+            </ul>
+            <hr
+              className='govuk-section-break govuk-section-break--xl govuk-section-break--visible'
+              aria-hidden
+            ></hr>
+
+            {/* Claims list */}
+            <div className='govuk-grid-column-two-thirds'>
+              {/* <>{children}</> */}
```

Comment:
same here

Reply:
The code here is temporarily that's why I need to leave the comments for future updates ..Thanks